s3tests_boto3/functional/test_s3select.py

```diff
@@ -360,42 +567,130 @@ def test_like_expressions():
     bucket_name = "test"
     upload_csv_object(bucket_name,csv_obj_name,csv_obj)
 
-    res_s3select_in = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name,'select count(*) from stdin where _1 like "%aeio%";')).replace("\n","")
+    res_s3select_like = remove_xml_tags_from_result(  run_s3select(bucket_name,csv_obj_name,'select count(*) from stdin where _1 like "%aeio%" ;')).replace("\n","")
+
+    x = bool(re.search("^.*failure.*$", res_s3select_like))
```

Comment:
it's more accurate to search for /failure -->/ at the beginning of the result.

this search command should embed into run_s3select (thus its transparent to all existing and future tests)